.github/workflows/ci.yml

```diff
@@ -222,7 +222,7 @@ jobs:
       - name: Install pnpm
         uses: pnpm/action-setup@v2.2.4
         with:
-          version: 7
+          version: 8.4.0
```

Comment:
I think we remove the version input value, what works at a global level between the different CI actions/steps is the `packageManager` field in the root `package.json` where it is specified as `pnpm@8.4.0`

Reply:
🙏

Reply:
https://github.com/dotansimha/graphql-yoga/pull/2793/commits/cf1fdf9dcb008972ecdbb41795d477471733c492